adapters/liftoff/liftoff.go

```diff
@@ -0,0 +1,180 @@
+package liftoff
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const SupportedCurrency = "USD"
+
+type adapter struct {
+	Endpoint string
+}
+
+type prebid struct {
+	Debug bool `json:"debug"`
+}
+
+type prebidExt struct {
+	Prebid prebid `json:"prebid"`
+}
+
+type liftoffImpressionExt struct {
+	*adapters.ExtImpBidder
+	// Ext represents the vungle extension.
+	Ext openrtb_ext.ImpExtLiftoff `json:"vungle"`
+}
+
+/* Builder */
```

Comment:
Please do not use this comment style. Go standards are to use two slashes followed by the function name and then the comment, such as:

```
// Builder builds a new instance of the Liftoff adapter for the given bidder with the given config.
```